modules/ROOT/pages/branded_ios_app/publishing_ios_app_10.adoc

```diff
@@ -0,0 +1,52 @@
+= Additional Server Configuration 
+:apple-universal-link-support-url: https://developer.apple.com/library/content/documentation/General/Conceptual/AppSearch/UniversalLinks.html
+:admin_manual: https://doc.owncloud.com/server/administration_manual/index.html
+:apple-app-site-association-file-url: https://developer.apple.com/documentation/security/password_autofill/setting_up_an_app_s_associated_domains
+
+== Add Support for Apple Universal Links
+
+There's some special changes that need to be made. 
+Quoting from Apple's official documentation on {apple-universal-link-support-url}[Universal Link Support]:
+
+[quote]
+____
+Adding support for universal links is easy. 
+There are three steps you need to take:
+
+. Create {apple-app-site-association-file-url}[an apple-app-site-association file] that contains JSON data about the URLs that your app can handle.
+. Upload the apple-app-site-association file to your HTTPS web server. You can place the file at the root of your server or in the `.well-known` subdirectory.
+. Prepare your app to handle universal links. 
+____
+
+The `apple-app-site-association` will be generated by ownbrander, and needs to be served statically on your site through HTTPS. 
+You can safely place it on the root folder of your ownCloud installation (e.g., `/var/www/owncloud`). 
+To achieve the second requirement, some changes will also need to be completed on your Apache configuration.
+
+If you configured your installation with the official {admin_manual}[Admin Manual], your `owncloud.conf` file should include the following:
```

Comment:
I propose to add a `=== Apache` section and below a `=== nginx` section.
I will add on a second comment the nginx info.

Reply:
I've made those changes. Just waiting on your NGINX configuration, and I'll add that in as well. Unless you're adding that yourself, locally.